src/sentry/api/endpoints/organization_member_details.py

```diff
@@ -71,6 +99,20 @@ def _is_only_owner(self, member):
 
         return True
 
+    def get(self, request, organization, member_id):
```

Comment:
is this endpoint returning anything specific to the member with `member_id`? or is it just returning data based on org / the signed in user?

Reply:
huh, you're right :/ pushing a fix

Reply:
i made it use member_id

Reply:
so, this returns the roles a given user has permission to grant, right? I think that this is going to get weird when we need to convert the member settings page, because that is going to need this endpoint to return a serialized version of a member -- meaning what role they are, what teams they're on, etc. 

This maybe should just be its own "roles" endpoint where only the get is implemented? Let's talk about it tomorrow.

Reply:
per our discussion, let's just make this a more generic endpoint that returns a serialized member (which can include `allowed_roles` or whatever you decide to call it)